src/main/java/com/benmorant/game/holycraap/controller/PeopleController.java

```diff
@@ -0,0 +1,70 @@
+package com.benmorant.game.holycraap.controller;
+
+import com.benmorant.game.holycraap.model.business.PeopleService;
+import com.benmorant.game.holycraap.model.entity.People;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Collection;
+import java.util.Optional;
+import javax.validation.Valid;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api")
+public class PeopleController {
+
+  private final Logger log = LoggerFactory.getLogger(PeopleController.class);
```

Comment:
You should make that static. No need to create one `Logger` per `PeopleController` instance (though there won't be many.

Also, final class/static attribute names are capitalized in Java (_e.g._ `LOG`).

Reply:
yes, indeed